Translate english into chinese for mouse.zh-cn.md of actons-api

geekmister opened the pull request:
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.


<!--- Provide a general summary of your changes in the Title above -->

### Description
Translate english into chinese for mouse.zh-cn.md of actons-api

### Motivation and Context
It solves the inconvenience of reading English for some Chinese engineers.

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->

geekmister committed:
Update mouse.zh-cn.md

Translate english into chinese for mouse.zh-cn.md of actons-api

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=1562) <br/>All committers have signed the CLA.

geekmister commented:
> ### 👷 Deploy request for _selenium-dev_ pending review.
> [Visit the deploys page to approve it](https://app.netlify.com/sites/selenium-dev/deploys)
> 
> Name	Link
> 🔨 Latest commit	[a57ea21](https://github.com/SeleniumHQ/seleniumhq.github.io/commit/a57ea21b02339cb1662831554a3d7d2ecfaad98f)

https://65af75c3a99ce4b4d7649cd9--polite-jelly-cc0866.netlify.app/zh-cn/documentation/webdriver/actions_api/mouse/

geekmister committed:
Merge branch 'trunk' into trunk

alaahong commented on website_and_docs/content/documentation/webdriver/actions_api/mouse.zh-cn.md:
```diff
@@ -185,11 +185,11 @@ This method combines moving to the center of an element with pressing and releas
-This is otherwise known as "hovering."
-Note that the element must be in the viewport or else the command will error.
+这个方法是将光标移动到元素的中心点。
+另一种叫法“悬浮”。
```
How about “悬停”？

alaahong replied:
refer: https://developer.mozilla.org/zh-CN/docs/Web/CSS/:hover

geekmister replied:
I think "悬停" or "悬浮" are both ok, because of some people call it "悬停" or "悬浮".

alaahong commented on website_and_docs/content/documentation/webdriver/actions_api/mouse.zh-cn.md:
```diff
@@ -15,10 +15,10 @@ There are only 3 actions that can be accomplished with a mouse:
 Selenium provides convenience methods that combine these actions in the most common ways.
 
-## Click and hold
+## 按下鼠标左键
```
how about “按住”？ otherwise seems no diff than click

geekmister replied:
Of course, "按住" is accuracy!

alaahong commented on website_and_docs/content/documentation/webdriver/actions_api/mouse.zh-cn.md:
```diff
@@ -302,10 +296,9 @@ the current mouse position.
 {{< /tabpane >}}
 
-## Drag and Drop on Element
+## 拖动和放下元素
```
How about "拖放"？
refer： https://developer.mozilla.org/zh-CN/docs/Web/API/HTML_Drag_and_Drop_API

geekmister replied:
Of course, "拖放" is accuracy!

alaahong reviewed (commented):
Thanks for your commits, just leave some comments on the words, can you take a view again?

geekmister committed:
Update mouse.zh-cn.md

Modify some words in accordance with the auditor's comments and translate 3 lines in the top.

geekmister reviewed (commented):
Reviewed. Need auditor review.

alaahong committed:
Merge branch 'trunk' into trunk

alaahong reviewed (approved):
Thanks for your updating, LGTM
